ci/docker/docker-in-docker-image/publish-docker-image/requirements.txt

```diff
@@ -1,4 +1,4 @@
-pyyaml==5.1
+pyyaml==5.4
```

Comment:
Without rebuilding the image can be broken now. 
There is no guarantee from semver.